File: python/meteroid/models/plan_version_summary.py
# this file is @generated
import dataclasses
from datetime import datetime

from ..serialization import BaseModel
from .plan_version_id import PlanVersionId


@dataclasses.dataclass
class PlanVersionSummary(BaseModel):
    created_at: datetime

    currency: str

    id: PlanVersionId

    is_draft: bool

    version: int

File: go/plan_version_summary.go
// this file is @generated
package meteroid

import "time"

type PlanVersionSummary struct {
	// RFC 3339 timestamp.
	CreatedAt time.Time `json:"created_at"`

	Currency string `json:"currency"`

	Id PlanVersionId `json:"id"`

	IsDraft bool `json:"is_draft"`

	Version int32 `json:"version"`
}
